symfony.lock

```diff
@@ -0,0 +1,262 @@
+{
+    "alecrabbit/php-cli-snake": {
```

Comment:
This is what I'm talking about. Why do we need this if we have composer?

Reply:
erm...

```bash
composer why -t alecrabbit/php-cli-snake                                                                                               
alecrabbit/php-cli-snake 0.5.0 Lightweight cli spinner with zero dependencies
└──seregazhuk/php-watcher v0.5.2 (requires alecrabbit/php-cli-snake ^0.5)
   └──drift/server 0.1.7 (requires seregazhuk/php-watcher ^0.5.2)
      └──drift/skeleton dev-symfony-flex (requires drift/server 0.1.*, >=0.1.7)
```

Reply:
I'm not talking about this line. I'm talking about the entire file, sorry.
I didn't find a way to comment an entire file.

Reply:
okay, sorry.